### PR TITLE
chore(BUX-223): adjust endpoints to new bux-server router

### PR DESCRIPTION
**Pull request by pawellewandowski98**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/BuxOrg/js-buxclient/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/BuxOrg/js-buxclient/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

# Description

- Adjusted methods to new bux-server router. Moved method which creates pub to Admin section and removed paymail duplicates

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -294,10 +294,10 @@ class BuxClient implements TransportService {
    * Admin only: Delete a paymail
    *
    * @param address string Paymail address (ie. example@bux.org)
-   * @return {PaymailAddress}
+   * @return void
    */
-  async AdminDeletePaymail(address: string): Promise<PaymailAddress> {
-    return await this.client.httpTransport.AdminDeletePaymail(address);
+  async AdminDeletePaymail(address: string) {
```
Comment (Nazarii-4chain) [inline src/index.ts:299]: Shouldn't we sent some confirmation that paymail has been successfully deleted ?

Comment (pawellewandowski98) [reply src/index.ts:299]: if there is no error, paymail has been deleted

**Review by dorzepowski — APPROVED** [review]

**Review by wregulski — APPROVED** [review]

**Review by Nazarii-4chain — APPROVED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -293,10 +293,10 @@ class SpvWalletClient implements TransportService {
   /**
    * Admin only: Delete a paymail
    *
-   * @param address string Paymail address (i.e. alias@example.com)
-   * @return {PaymailAddress}
+   * @param address string Paymail address (ie. example@bux.org)
```
Comment (Nazarii-4chain) [inline src/index.ts:296]:
```suggestion
   * @param address string Paymail address (ie. example@spv-wallet.org)
```

File: src/index.ts [hunk 3]
```diff
@@ -369,6 +369,17 @@ class SpvWalletClient implements TransportService {
     return await this.client.httpTransport.AdminGetXPubsCount(conditions, metadata);
   }
 
+  /**
+   * Admin only: Register a new xPub into the Bux server
```
Comment (Nazarii-4chain) [inline src/index.ts:373]:
```suggestion
   * Admin only: Register a new xPub into the SPV Wallet
```

File: src/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (pawellewandowski98) [reply src/index.ts]: changed to your proposal

Comment (pawellewandowski98) [reply src/index.ts]: changed

**Review by chris-4chain — APPROVED** [review]

**Comment by chris-4chain** [comment]
testing 🧪

**Review by chris-4chain — APPROVED** [review]

**Pull request merged**